nemo_skills/mcp/servers/plane_tickets.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Example motivated by https://github.com/modelcontextprotocol/python-sdk/blob/main/examples/snippets/servers/streamable_starlette_mount.py
+from __future__ import annotations
+
+import uuid
+from typing import Any
+import contextlib
+
+from starlette.applications import Starlette
+from starlette.routing import Mount
+from mcp.server.fastmcp import FastMCP
+
+"""
+Run with:
+    uvicorn nemo_skills.mcp.servers.plane_tickets:app --reload --port 8003
+"""
+
+# Initialize FastMCP server
+mcp = FastMCP(name="plane_tickets", stateless_http=True)
```

Comment:
just to double check, the plan is to have this as part of the sandbox, right? So it will keep its current api, but also add the new endpoints for tools? The current examples use stdio for communication, which I think we won't actually need for the initial implementation, but instead we should test an http protocol there